site/content/contribute/gsoc/_index.md

```diff
@@ -0,0 +1,223 @@
+
+However, as of today, plugins are not able to modify the input textbox, so the first and most important task in this project would be to change the plugin API to add such a feature. These architectural changes are valuable in themselves, so a project that would only deliver this would be considered successful. However, if the student finishes this task before the coding period ends, the next task would be to actually implement the quote plugin, using the new functionality as a basis for it.
+
+-   **Expected outcomes**: The infrastructure changes for letting plugins modify the input box. If there is time, a new plugin that adds the quoting feature.
+-   **Skills required**: React
+-   **Difficulty**: Medium
+-   **Mentors**: Daniel Espino García, Alejandro García Montoro
+
+### Timed reminders on specific posts
+
+It's usual that we read an interesting message but don't have time to process it at the moment. Some of us use the "Mark as unread" feature or the TODO plugin for this, but it would be very valuable to have a feature that reminds us of that specific message in some time, generating a new notification in 30 minutes, one hour, or the next day.
+
+This project is focused around that specific feature, and the goal is to build a new plugin that adds a new entry to the post menu for selecting a time at which the user will be reminded of that specific message. The community has already started development on this, but it was never finished, so this is an opportunity to bring back that work and take it to the finish line, better defining its scope and usability in the process.
+
+-   **Expected outcomes**: A new plugin to add timed reminders to posts.
+-   **Skills required**: React, Go
+-   **Difficulty**: Easy
+-   **Mentors**: Alejandro García Montoro, Eric Sethna
+
+### Active call banner
+
+There are several plugins in Mattermost that integrate the product with video conference software such as Jitsi or Zoom. When a user starts a meeting, a message is automatically posted to the channel inviting other users to join. This post is then edited when the meeting ends, displaying the length of the meeting. However, the plugin post can quickly be buried in the channel if other messages are posted.
+
+This project tries to overcome this problem by adding an always visible banner to the channels where there is an active meeting.
+
+The banner should be displayed when there's an active video call, either in Jitsi or Zoom. Clicking the banner would let the user automatically join the call, and it can display information about the meeting itself: the number of participants, current run time, whether it's in Jitsi or Zoom. This idea should be discussed with the UX team to come up with a design that could be similar to the "x new messages" that appears in an unread channel.
```

Comment:
```suggestion
The banner should be displayed when there's an active video call, either in Jitsi or Zoom. Clicking the banner would let the user automatically join the call, and it would display information about the meeting itself: the number of participants, current run time, and whether it's in Jitsi or Zoom. This idea should be discussed with the UX team to come up with a design that could be similar to the "x new messages" that appears in an unread channel.
```